docs/src/tutorials/transition.md

```diff
@@ -0,0 +1,52 @@
+# Transition from `GroupManifolds` in `Manifolds.jl`
+
+One predecessor of `LieGroups.jl` are the [`GroupManifold`](@extref `Manifolds.GroupManifold`)s in `Manifolds.jl`.
+While this package provides the same features, one reason for a new package is,
+that a “restart” offers the opportunity to put the main focus for the functions in this package
+really on Lie groups.
+
+This tutorial provides an overview of the necessary changes to your code if you based it on the predecessor.
+
+## Table of function names and its successors
+
+The following table lists all functions related to `GroupManifolds` and their new names
+or replacements here in `LieGroups.jl`. In this code `G` always refers to the `GroupManifold`
+in the first column and the [`LieGroup`](@ref) in the second.
+Lie group elements (points) are always `g,h`,
+Lie algebra elements (vectors) always `X, Y`.
+
+New functions and types in this package are only mentioned, if they are worth a comment and if something changed.
+
+The list is alphabetical, but first lists types, then functions
+
+| `Manifolds.jl` | `LieGroups.jl` | Comment |
+|:---------- |:---------- |:-------------- |
+| `AdditionOperation` | [`AdditionGroupOperation`](@ref) | |
+| `LeftForwardAction` | [`LeftGroupOperation`](@ref)
+| `RightBackwardAction` | [`RightGroupOperation`](@ref) | |
+| `LeftBackwardAction` | [`InverseRightGroupOperation`](@ref) | note that this is now also aa [`AbstractLeftGroupActionType`](@ref) |
+| | [`LieAlgebra`](@ref)`(G)` | new alias to emphasize its manifold- and vector structure as well as for a few dispatch methods. |
+| `GroupManifold(M, op)` | [`LieGroup`](@ref)`(M, op)` | |
+| `RightForwardAction` | [`InverseLeftGroupOperation`](@ref) | note that this is an [`AbstractRightGroupActionType`](@ref) |
+| `adjoint` | [`adjoint`](@ref) | now implemented with a default, when you provide [`diff_conjugate!`](@ref).
```

Comment:
Was this `adjoint_action`?

Reply:
No, this was really just `adjoint`; this table does not yet cover much with actions, since actins have not yet been done (much) in this package, besides a first few thoughts on the interface.

Reply:
See also https://juliamanifolds.github.io/LieGroups.jl/previews/PR12/interface/group/#Base.adjoint-Tuple{LieGroup,%20Any,%20Any} (but ignore the last sentence, I have to move that still).

Reply:
> See also [https://juliamanifolds.github.io/LieGroups.jl/previews/PR12/interface/group/#Base.adjoint-Tuple{LieGroup,%20Any,%20Any}](https://juliamanifolds.github.io/LieGroups.jl/previews/PR12/interface/group/#Base.adjoint-Tuple%7BLieGroup,%20Any,%20Any%7D) (but ignore the last sentence, I have to move that still).

ok, I thought it was the adjoint_action because it matches https://juliamanifolds.github.io/Manifolds.jl/stable/manifolds/group.html#Manifolds.adjoint_action-Tuple{AbstractDecoratorManifold,%20Any,%20Any,%20Any}
From what I understand:
Wikipedia refers to this as the [adjoint representation or adjoint action](https://en.wikipedia.org/wiki/Adjoint_representation).
 "Stochastic Models, Information Theory, and Lie Groups, Volume 2" calls it the adjoint operator Ad(g).

Reply:
Well, the `adjoint` here is a special case of `adjoint_action` if you want to be precise, namely when you consider the Lie groups group operation $\circ$ as a special case of a Lie group action, where the Lie group acts on itself (that is the manifold G acts on is M=G itself).
This is also the only group action that is currently available, since I want to have a few Lie groups and their operations finished before diving (back) into actions.

All of `adjoint`, `conjugate` and its `diff_conjugate` are new functions here, that I first and foremost needed for understanding, but that will also hopefully be useful later in defining a few new things. So maybe the answer to the initial question is maybe more like Nes or Yo ;)

Reply:
My reference (hopefully used often) is Hilgert&Neeb https://link.springer.com/book/10.1007/978-0-387-84794-8 – mainly because a colleague of mine recommended that and I felt it was easy to read and reference to for me.

Reply:
Ok, thank you.  I'll wait for the actions pr.  I'm tracking `adjoint_action` since I'm currently using it and figuring out a few related things.

Reply:
Great! It might still take a while, since I would first like to make sure the groups and their group operations are safe and sound in their interface before continuing. but for that PR I have then in mind to ping you. The `adjoint_action` name will stay as before for the function name for sure.